### ISSUE TITLE
Add GitHub Action based CI

### ISSUE THREAD

**Issue by aentinger** [arduino-libraries/Arduino_MKRIoTCarrier#22]
And remove hardcoded build badge to obsolete travis CI.

**Comment by marqdevx** [comment]
Added, thanks!

**Comment by aentinger** [comment]
Great @aentinger :rocket: Please next time try out your changes in a separate pull request and merge only after everything works. You can reach out to me or @per1234 (also via Slack) if you want to pick our brains ;)
